components/webui/imports/ui/IngestView/panels/IngestionJobs/IngestionJobRow.jsx

```diff
@@ -37,19 +40,32 @@ const COMPRESSION_JOB_STATUS_ICONS = Object.freeze({
  * @return {React.ReactElement}
  */
 const IngestionJobRow = ({job}) => {
-    let speedText = "";
     let uncompressedSizeText = "";
+    let speedText = "";
     let compressedSizeText = "";
 
     if (null === job.duration && null !== job.start_time) {
         job.duration = dayjs.duration(
             dayjs() - dayjs(job.start_time)
         ).asSeconds();
     }
-    if (0 < job.duration) {
+
+    const uncompressedSize = Number(job.uncompressed_size);
+    if (false === isNaN(uncompressedSize) && 0 !== uncompressedSize) {
+        uncompressedSizeText = computeHumanSize(uncompressedSize);
+    }
+
+    if (
+        false === isNaN(uncompressedSize) &&
+        0 !== uncompressedSize &&
+        0 < job.duration
+    ) {
         speedText = `${computeHumanSize(job.uncompressed_size / job.duration)}/s`;
-        uncompressedSizeText = computeHumanSize(job.uncompressed_size);
-        compressedSizeText = computeHumanSize(job.compressed_size);
+    }
+
+    const compressedSize = Number(job.compressed_size);
```

Comment:
can we put compressedSize and uncompressedSize close to each other instead of being separated by speedText?